lib/peg.d.ts

```diff
@@ -885,14 +917,25 @@ export interface BuildOptionsBase {
 }
 
 export interface ParserBuildOptions extends BuildOptionsBase {
-  /** If set to `"parser"`, the method will return generated parser object; if set to `"source"`, it will return parser source code as a string (default: `"parser"`) */
+  /**
+   * If set to `"parser"`, the method will return generated parser object;
+   * if set to `"source"`, it will return parser source code as a string
+   * (default: `"parser"`)
+   */
```

Comment:
I think I like the "source-and-map" API approach in your follow-on.

Reply:
Ok, I'll include that commit here